platform/android/export/export.cpp

```diff
@@ -2225,25 +2227,34 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 			return ERR_CANT_CREATE;
 		}
 
-		if (ep.step("Verifying APK...", 104)) {
+		if (ep.step("Verifying " + export_label + "...", 104)) {
 			return ERR_SKIP;
 		}
 
 		args.clear();
 		args.push_back("-verify");
 		args.push_back("-keystore");
 		args.push_back(keystore);
-		args.push_back(apk_path);
+		args.push_back(export_path);
 		args.push_back("-verbose");
 
 		OS::get_singleton()->execute(jarsigner, args, true, NULL, NULL, &retval);
 		if (retval) {
-			EditorNode::add_io_error("'jarsigner' verification of APK failed. Make sure to use a jarsigner from OpenJDK 8.");
+			EditorNode::add_io_error("'jarsigner' verification of " + export_label + " failed. Make sure to use a jarsigner from OpenJDK 8.");
 			return ERR_CANT_CREATE;
 		}
 		return OK;
 	}
 
+	void _clear_assets_directory() {
+		DirAccess *da_res = DirAccess::create(DirAccess::ACCESS_RESOURCES);
```

Comment:
I think you need to `memdelete` the DirAccess once you're done with it, or use DirAccessRef which should free it when going out of scope.

Reply:
Done!